src/kaocha/type/clojure/spec/alpha/check.clj

```diff
@@ -0,0 +1,32 @@
+(ns kaocha.type.clojure.spec.alpha.check
+  (:require [clojure.spec.alpha :as s]
+            [kaocha.hierarchy :as hierarchy]
+            [kaocha.load :as load]
+            [kaocha.testable :as testable]
+            [kaocha.type :as type]
+            [kaocha.type.clojure.spec.alpha.fdef :as type.fdef]
+            [kaocha.type.clojure.spec.alpha.ns :as type.spec.ns]))
+
+(alias 'stc 'clojure.spec.test.check)
+
+(defmethod testable/-load :kaocha.type/clojure.spec.test.alpha.check
+  [{::keys [syms] :as testable}]
+  (-> (if (= syms :all-fdefs)
+        (load/load-namespaces testable :kaocha/source-paths type.spec.ns/->testable)
+        (assoc testable :kaocha.test-plan/tests (type.fdef/load-testables syms)))
+      (testable/add-desc "clojure.spec.alpha.check")))
```

Comment:
How come this testable type does not have a `-run` method?

Reply:
Just haven't gotten that far yet. I think the implementation will be nearly the same as `:kaocha.type/clojure.test` so I probably want to reuse it outright, don't I?